src/app/metadata/features/search-result-list/search-result-list.component.html

```diff
@@ -1,7 +1,7 @@
-<div class="bg-primary mb-2 grid w-full grid-cols-[14rem_1fr] px-6 py-2 text-white">
+<div class="bg-primary mb-4 grid w-full grid-cols-[14rem_1fr] px-6 py-2 text-white">
   <span>Dataset ID</span><span>Title</span>
 </div>
-<div [class.invisible]="!loading()" class="mb-3">
+<div [class.hidden]="!loading()" class="mb-3">
```

Comment:
I think the "invisible" was intentional, so that after loading the layout would not change (the results would appear exactly where the skeletons were).

Reply:
This should only be the loading bar? The problem is with invisible, the div causes spacing issues.

Reply:
Can you fix this in a separate PR?